addons/mail/static/src/core/sound_effects_service.js

```diff
@@ -72,6 +73,60 @@ export class SoundEffects {
             }
         }
     }
+    /**
+     * handle stop and start on voice messages
+     * @param {import("@mail/rtc/voice_analyser").VoiceAnalyser} analyser
+     */
+    onClickAnalyser(analyser) {
+        if (
+            this.activeAnalyser &&
+            this.activeAnalyser !== analyser &&
+            this.activeAnalyser.state.playing
+        ) {
+            this.activeAnalyser.soundEffectsService.onClickAnalyser(this.activeAnalyser);
+        }
+        this.activeAnalyser = analyser;
+        if (analyser.state.playing) {
+            clearInterval(analyser.state.intervalId);
+            analyser.state.paused = true;
+            analyser.state.playing = false;
+            Promise.resolve(analyser.audioRef.el.pause()).catch(() => {
+                analyser.state.playing = true;
+                analyser.state.paused = false;
+            });
+            return;
+        }
+
+        if (analyser.state.paused) {
+            analyser.SetIntervalId();
+            analyser.state.playing = true;
+            analyser.state.paused = false;
+            Promise.resolve(analyser.audioRef.el.play()).catch(() => {
+                analyser.state.paused = true;
+                analyser.state.playing = false;
+            });
+            requestAnimationFrame(analyser.renderFrame.bind(analyser));
+            return;
+        }
+
+        analyser.audioRef.el.load();
+        if (!analyser.analyser) {
+            var context = new AudioContext();
+            var src = context.createMediaElementSource(analyser.audioRef.el);
+            analyser.analyser = context.createAnalyser();
+            src.connect(analyser.analyser);
+            analyser.analyser.connect(context.destination);
```

Comment:
`analyser.analyser` (AnalyserNode) inherits from AudioNode and should also be [disconnected](https://developer.mozilla.org/en-US/docs/Web/API/AudioNode/disconnect)